eiffel-vocabulary/EiffelArtifactDeployedEvent.md

```diff
@@ -0,0 +1,19 @@
+
+# EiffelArtifactDeployedEvent (ArtD)
+
+The EiffelArtifactDeployedEvent declares that a software artifact (declared by EiffelArtifactCreatedEvent) has been deployed and is consequently available for use.
+
+## Data Members
+### data.reason
+__Type:__ String  
+__Required:__ No  
+__Description:__ The location at which the artifact is deployed.
```

Comment:
This doesn't make any sense. Is it a reason or a location? And I would have thought we'd use an ENVIRONMENT reference for the location?